src/open_inwoner/configurations/tests/test_colors.py

```diff
@@ -27,6 +27,10 @@ def test_contrast_is_checked(self):
         form["secondary_font_color"] = "#FFFFFF"
         form["accent_color"] = "#FFFFFF"
         form["accent_font_color"] = "#FFFFFF"
+        # django-jsonform requires JS to work properly and with Webtest the default
+        # value for ArrayFields is an empty string, causing it crash to when trying to parse
+        # that value as JSON
+        form["recipients_email_digest"] = "[]"
```

Comment:
Unfortunately jsonform doesn't work with webtest out of the box

Reply:
Ugh, well at least the comment is explicit about it.